### PR TITLE
new playbook: oh-my-zsh integration

### PR DESCRIPTION
**Pull request by paterik**
I've added a new playbook for the well known oh-my-zsh terminal/zsh extension to your playbook portfolio. Please check my contribution and let me know if there is any open issue / problem within my code.

best regards,

Patrick

### CODE REVIEW

File: roles/libreoffice/tasks/install.yml [hunk 1]
```diff
@@ -42,17 +42,17 @@
...
   vars:
-    qry: 'http://download.documentfoundation.org/libreoffice/.*/[.0-9]+/deb/x86_64/LibreOffice_[.0-9]+_Linux_x86-64_deb.tar.gz'
+    qry: 'download.documentfoundation.org/libreoffice/.*/[.0-9]+/deb/x86_64/LibreOffice_[.0-9]+_Linux_x86-64_deb.tar.gz'
   check_mode: no
   when: libreoffice_installed.rc == 1
 
 - name: ensure libreoffice is downloaded
   unarchive:
-    src: "{{ libreoffice_download_link }}"
+    src: "https://{{ libreoffice_download_link_base }}"
```
Comment (cytopia) [inline roles/libreoffice/tasks/install.yml:55]: Thanks for fixing :+1:

File: roles/oh-my-zsh/default/main.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+
+# 'install', 'remove' or 'ignore'
+oh_my_zsh: 'ignore'
```
Comment (cytopia) [inline roles/oh-my-zsh/default/main.yml:4]: Can you add:

```
oh_my_zsh: 'install'
```
in 

* `host_vars/debian-stretch.yml`: https://github.com/cytopia/ansible-debian/blob/master/host_vars/debian-stretch.yml#L92
* `host_vars/debian-testing.yml`: https://github.com/cytopia/ansible-debian/blob/master/host_vars/debian-testing.yml#L92

The two debian host vars are the generic ones and used for the travis checks.

**Review by cytopia — COMMENTED** [review]
Thanks for the PR. Can you address the two changes in order for travis to properly check it.

File: roles/oh-my-zsh/default/main.yml (hunk 2 above)
Comment (paterik) [reply roles/oh-my-zsh/default/main.yml:4]: no problem ... required changes are made.

**Comment by cytopia** [comment]
@paterik you can also run the tests locally inside the shippped Docker container:
```bash
docker build -t ansible-debian .
docker run --rm -e MY_HOST=debian-stretch -e tag=oh-my-zsh -t ansible-debian
```

**Comment by paterik** [comment]
ahh ... thank you for the tip (it seems to be working now) :)

**Comment by paterik** [comment]
Travis-jobs 7 & 10 on latest build (427) are still failing, can't find out why (raw logs shown no errors beside the final exit code 1) thats strange ... may you could restart the build on travis and check the results by your own? ... sorry about the inconvenience :-/

**Comment by cytopia** [comment]
Looks like the idempotence tests failed (2nd run) for `cytopia` profiles. No idea why either. I've restarted both jobs. However that shouldn't be connected to your changes.

**Pull request merged**